.travis.yml

```diff
@@ -2,9 +2,12 @@
 ---
 language: python
 env:
+# tuned cannot run in a container e.g. pytest unit tests of the module will fail
+# so use RUNNING_IN_CONTAINER=true so some tests can be skipped
   global:
     - LSR_ANSIBLES='ansible==2.6.* ansible==2.7.* ansible==2.8.*'
     - LSR_MSCENARIOS='default'
+    - RUNNING_IN_CONTAINER=true
```

Comment:
Travis tests run in a container? (Molecule uses containers, but where are containers being used in Travis outside Molecule?)

Reply:
Maybe I don't understand how travis works then - because I was able to reproduce the error I see in travis when it runs pytest, by running pytest locally in a ubuntu xenial container.

Reply:
@tyll @i386x can you help please? I am not that familiar with Travis myself.

Reply:
@richm do you see the error also when you run in a xenial VM? The default for Travis nowadays is to run a VM not a container. Therefore only molecule should be using containers at the moment.